docs/external-listener/index.md

```diff
@@ -59,17 +59,15 @@ To configure an external listener that uses the LoadBalancer access method, comp
           accessMethod: LoadBalancer
     ```
 
-1. Configure an *ingress controller* in the `ingressController` field of the `KafkaCluster` custom resource. For example:
+1. Set the ingress controller. The ingress controllers that are currently supported for load balancing are:
 
-    ```yaml
-    spec:
-      ingressController: "envoy"
-    ```
+    - `envoy`: uses Envoy proxy as an ingress.
+    - `istioingress`: uses Istio proxy gateway as an ingress. Istio ingress is the default controller for Kafka clusters provisioned with [Supertubes](/docs/supertubes/overview/), since those clusters run inside an Istio mesh.
```

Comment:
```suggestion
    - `istioingress`: uses Istio proxy gateway as an ingress. Istio ingress is the default controller for Kafka clusters provisioned with [SDM](/docs/supertubes/overview/), since those clusters run inside an Istio mesh.
```